packages/expo-status-bar/src/StatusBar.android.tsx

```diff
@@ -0,0 +1,92 @@
+import { SystemBars } from 'react-native-edge-to-edge';
```

Comment:
We have to import the `SystemBars` conditionally if `isEdgeToEdge` is `true`.  Otherwise importing `SystemBars` tries to import the turbomodule which is not autolinked, leading to errors.

Reply:
fixed in https://github.com/expo/expo/pull/36087/commits/e8d4e8afd8a4c1f4caf2ae2440e6c332882c5194